docs/ref/models/indexes.txt

```diff
@@ -30,16 +30,66 @@ options`_.
 
 .. attribute:: Index.fields
 
-A list or tuple of the name of the fields on which the index is desired.
+A list or tuple of the name of the fields or database expressions on which the index is desired.
```

Comment:
Please also consider adding examples of mixed column and function indexes in this section.